Andlet/Andlet/Models/PropertyModel.swift

```diff
@@ -10,7 +10,7 @@ import FirebaseFirestore
 
 // Hacer que PropertyModel conforme a Hashable y Equatable
 struct PropertyModel: Identifiable, Codable, Hashable, Equatable {
-    @DocumentID var id: String?
+    var id: String?
```

Comment:
why do we remove the `@DocumentID ` ?

Reply:
Because the way we are using the database don't allow us to make the models identifiable,